kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
Ok, changed that now. Should I change the [other occurrances](https://github.com/mozilla/kuma/search?utf8=%E2%9C%93&q=%22{{+_%28%27{title}%27%29|fe%28title%3Ddocument.title%29+}}%22&type=Code), too?

Sebastian